### DISCUSSION TITLE
hover description text on jobs and resources?

### DISCUSSION THREAD

**Discussion by x3nb63** [concourse/concourse#6719] [Ideas]
(I am quite sure this came up in the past, just can't find it)

So then, here we have a pipeline which is rather big and people staring at the WebUI feel the need to know what exactly a job or resource does or was supposed to do.

This far the first natural consequence of those people is to start a Wiki page that describes each and every job and resource in a table with a description field which then has a set of notes, a  list of bullet points actually, and even some URLs to other systems like a code viewer.

The second natural consequence of those people is to invent a really complex nomenclature that stuffs a lot of information into the job and resource names.

Now that wiki page needs maintenance whenever the pipelines are changed, while the names have a tendency towards the readability of IPv6 addresses.

Right now it occurs obvious to me that a solution would be to have `desc:` or `doc:` yaml attribute on the objects shown as boxes on the WebUI, to define notes and links. The WebUI would then pop a little hovering box or something to show it.

**Comment by chenbh** [comment]
Oh yeah this has been a much request feature, see: https://github.com/concourse/concourse/issues/6258 https://github.com/concourse/concourse/issues/2025 https://github.com/concourse/concourse/issues/2687 https://github.com/concourse/concourse/pull/6155. 

I think the current state is that we do want something like this, but we'll need to iron out the design and interface. It's also pretty low on our priorities right now so it'll most likely be up to the community to contribute this feature.

**Comment by x3nb63** [comment]
ok, I'll check if I am granted the time. (unlikely, cause Go would be new to me)

regarding this feature:  **tooltip** is probably the best description of what it should be; rather short referencing than long texting.

EDIT:  many jobs here put doxygen manuals and reports into buckets - a lot explaining would already be achieved by just linking to those ...
